packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -56,14 +45,13 @@ export class NativeService {
   private isInitMessage = (message: PostMeMessage): boolean =>
     message.message === strings.webviewIsRendered
 
-  public registerWebview = (webviewRef: WebviewRef): void => {
-    const uri = this.getUri(webviewRef)
+  public registerWebview = (uri: string, ref: WebviewRef): void => {
```

Comment:
new API

Reply:
this introduces a breaking change 😢 

why not using either

```suggestion
  public registerWebview = ({uri, ref}): void => {
```
or
```suggestion
  public registerWebview = (ref: WebviewRef, uri: string): void => {
  if(!uri) {
    // should do something
  }
```

Reply:
I think it's better this way because uri is always the first argument. Also these methods are only called once in the entire cozy code right now so it's ok I think